articles/connections/passwordless/sms-gateway.md

```diff
@@ -72,32 +80,48 @@ This will return your SMS connection with your Twilio settings:
         "time_step": 300,
         "length": 6
       }
-    }
+    },
+    "is_domain_connection": false,
+    "enabled_clients": []
 }
 ```
 
-3. Send the updated configuration to the Management API using the [PATCH connections endpoint](/api/v2#!/Connections/patch_connections_by_id).
+    If your SMS Gateway accepts authenticated requests that are token-based, you can add the following to your `options` object:
 
-After updating the connection for any user that signs up or authenticates using the <dfn data-key="passwordless">Passwordless</dfn> SMS connection, the following payload will be sent to your SMS gateway:
+```json
 
+"gateway_authentication": {
+    "method": "bearer",
+    "subject": "urn:Auth0",
+    "audience": "urn:MySmsGateway",
+    "secret": "MySecretToSignTheToken"
+}
 ```
+
+    With this configuration, a JWT will be added to the `Authorization` header which contains a token with the `subject` and `audience` configured in the connection and signed with the `secret` when the payload is sent to your SMS Gateway.
+
+    If your secret is base64-url-encoded, set `options.gateway_authentication.secret_base64_encoded` to `true`.
+
+4. Once you have updated your connection, Auth0 will send the following to your SMS Gateway every time a user signs up or logs in with your Passwordless connection.
+
+```json
 {
   "recipient": "+1 399 999",
   "body": "Your verification code is: 12345",
   "sender": "+1 234 567"
 }
 ```
 
-If you set the `forward_req_info` property to **true**, the gateway will also receive information from the HTTP request that initiated the Passwordless process. This includes the IP address of the client calling `/passwordless/start` and its User Agent.
+If you set the `forward_req_info` property to `true`, the gateway will also receive information from the HTTP request that initiated the Passwordless process. This includes the IP address of the client calling `/passwordless/start` and its User Agent.
```

Comment:
Where? We've just been talking about authenticated requests and are switching out of that now. Might be good to point them to where they should include this. (It makes sense to me, but I can see someone making a mistake here.)